### ISSUE TITLE
total_variant_count smaller than window size?

### ISSUE THREAD

**Issue by lauterbur** [alek0991/iSAFE#7]
Dear Ali,

I am running into an issue that iSAFE is breaking with certain input simulations. I think I've tracked this down to the create_rolling_indices function in isafeclass.py, and that it occurs when total_variant_count is smaller than w_start + w_size. What would you recommend in this case?

The full command line and error are:
```
python2 /mnt/stat_scripts/iSAFE-1.0/src/isafe.py --format hap -i ${SIMPATH}/${OUTNAME}.hap.iSAFE -o test --MaxFreq 1 --MinRegionSize-bp 49000 --MinRegionSize-ps 1 --IgnoreGaps

/mnt/stat_scripts/iSAFE-1.0/src/isafe.py:125: UserWarning: With [--format hap], iSAFE assumes that derived allele is 1 and ancestral allele is 0 in the input file, and the selection is ongoing (the favored mutation is not fixed).
  warnings.warn("With [--format hap], iSAFE assumes that derived allele is 1 and ancestral allele is 0 in the input file, and the selection is ongoing (the favored mutation is not fixed).")
/mnt/stat_scripts/iSAFE-1.0/src/isafe.py:180: UserWarning: Warning: There is 35 gaps with size greater than 10kbp.
  warnings.warn("Warning: There is %i gaps with size greater than %ikbp." % (num_gaps, args.MaxGapSize/ 1e3))
119 SNPs and 100 Haplotypes
[--Traceback (most recent call last):
  File "/mnt/stat_scripts/iSAFE-1.0/src/isafe.py", line 212, in <module>
    run()
  File "/mnt/stat_scripts/iSAFE-1.0/src/isafe.py", line 204, in run
    obj_isafe.fire(status=status)
  File "/mnt/stat_scripts/iSAFE-1.0/src/isafeclass.py", line 152, in fire
    self.snps_summary = creat_snps_information_df(self.windows_summary)
  File "/mnt/stat_scripts/iSAFE-1.0/src/isafeclass.py", line 47, in creat_snps_information_df
    return pd.concat(temp_list).reset_index(drop=True)
  File "/usr/lib64/python2.7/site-packages/pandas/core/reshape/concat.py", line 228, in concat
    copy=copy, sort=sort)
  File "/usr/lib64/python2.7/site-packages/pandas/core/reshape/concat.py", line 262, in __init__
    raise ValueError('No objects to concatenate')
ValueError: No objects to concatenate
```
The hap file associated with this error:
[neutral_data_26_c500000.hap.txt](https://github.com/alek0991/iSAFE/files/6266861/neutral_data_26_c500000.hap.txt)
Created from an ms run as follows:
`ms 100 1 -t 925.466843 -r 616.837564 -eN .00050 .02000`

Thanks very much,
Elise

**Comment by alek0991** [comment]
Dear Elise, 

Thank you for brining this bug to my attention. I just released version 1.0.7 to address this issue and few others. 

Please note that iSAFE's default parameters (like `--window, --step, --MinRegionSize-bp, --MinRegionSize-ps`, and others) are selected for human populations. In case iSAFE is applied to populations that are very different from human population (regarding their mutation rate, population size, or recombination rate) these parameter need to be tuned to maximize the performance of iSAFE. 

Please let me know if you have any question.
Best,
Ali

**Comment by lauterbur** [comment]
Hi Ali,

Thank you, that was fast. 

Cheers,
Elise
